packages-next/auth/src/gql/getBaseAuthSchema.ts

```diff
@@ -64,7 +56,11 @@ export function getBaseAuthSchema({
               itemPlural: list.adminUILabels.plural,
               code: result.code,
             });
-            return { code: result.code, message };
+            // We log the failure mode to the console, but do not
+            // expose this info via the graphQL interface.
+            // FIXME: Need to consider how to do server logging in general.
+            console.log({ code: result.code, message, identity: args[identityField] });
```

Comment:
We arguably shouldn't do this either; leaking data into production logs is mostly not safe

Reply:
Specifically, logging PII (which the identity field will often contain) is going to be a problem for anyone needing to comply with the GDPR. Although I suppose you could argue that, since auth failed, the value often won't be correct...? Regardless, might be better to exclude it.